Workbooks/Intune Audit/Audit/IntuneAuditActivity.workbook

```diff
@@ -0,0 +1,390 @@
+                  "createdTime": "2019-03-04T13:46:59.079Z",
+                  "isInitialTime": false,
+                  "grain": 1,
+                  "useDashboardTimeRange": false
+                },
+                {
+                  "durationMs": 86400000,
+                  "createdTime": "2019-03-04T13:46:59.079Z",
+                  "isInitialTime": false,
+                  "grain": 1,
+                  "useDashboardTimeRange": false
+                },
+                {
+                  "durationMs": 172800000,
+                  "createdTime": "2019-03-04T13:46:59.079Z",
+                  "isInitialTime": false,
+                  "grain": 1,
+                  "useDashboardTimeRange": false
+                },
+                {
+                  "durationMs": 259200000,
+                  "createdTime": "2019-03-04T13:46:59.079Z",
+                  "isInitialTime": false,
+                  "grain": 1,
+                  "useDashboardTimeRange": false
+                },
+                {
+                  "durationMs": 604800000,
+                  "createdTime": "2019-03-04T13:46:59.079Z",
+                  "isInitialTime": false,
+                  "grain": 1,
+                  "useDashboardTimeRange": false
+                },
+                {
+                  "durationMs": 1209600000,
+                  "createdTime": "2019-03-04T13:46:59.079Z",
+                  "isInitialTime": false,
+                  "grain": 1,
+                  "useDashboardTimeRange": false
+                },
+                {
+                  "durationMs": 2592000000,
+                  "createdTime": "2019-03-04T13:46:59.079Z",
+                  "isInitialTime": false,
+                  "grain": 1,
+                  "useDashboardTimeRange": false
+                },
+                {
+                  "durationMs": 5184000000,
+                  "createdTime": "2019-03-04T13:46:59.079Z",
+                  "isInitialTime": false,
+                  "grain": 1,
+                  "useDashboardTimeRange": false
+                },
+                {
+                  "durationMs": 7776000000,
+                  "createdTime": "2019-03-04T13:46:59.079Z",
+                  "isInitialTime": false,
+                  "grain": 1,
+                  "useDashboardTimeRange": false
+                }
+              ],
+              "allowCustom": true
+            },
+            "timeContextFromParameter": null
+          }
+        ],
+        "style": "pills",
+        "resourceType": "microsoft.resourcegraph/resources"
+      },
+      "conditionalVisibility": null
+    },
+    
+    {
+      "type": 1,
+      "content": {
+        "json": "## Audit Events Trend"
+      },
+      "conditionalVisibility": null
+    },
+    
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "IntuneAuditLogs\r\n| summarize count() by bin(TimeGenerated, {TimeRange:grain})",
```

Comment:
In general, if you have to bin by time, then make-series is a better alternative because it handles buckets with 0 items better. Summarize can cause chart to time-shift if the 0 items are at the beginning or end.

Reply:
Thank you Arun. I will play with make-series and follow up with another checkin.